### ISSUE TITLE
[Bug] Crash running MapRouletteUpload

### ISSUE THREAD

**Issue by atiannicelli** [osmlab/atlas-checks#415]
**Describe the bug**
Crash seen while running mapRouletteUpload when any log files in the output folder have lines without a "features" key.

**To Reproduce**
1. Produce a log file with WaterAreaCheck
2. Then execute the mapRouletteUpload command against that log file (or while that log file is in the output directory.
3. You should see the crash below

**Expected behavior**
1) I believe that WaterAreaCheck should no be producing flags without any flagged objects. So this can be better.
2) If a flag exists in the log file without a "features" key then the mapRouletteUpload command should not crash. 

**Screenshots & Stack traces**
Command executed:
```
java -cp build/libs/atlas-checks-*-SNAPSHOT-shaded.jar org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand -maproulette='https://maproulette.org:443:Alex_testing_123:api|key' -logfiles='output/flag' -outputPath='output/' -config='config/configuration.json' -checkinComment='#AtlasChecks' -countries='NZL' -checks='DuplicateWaysCheck'
```

Stack trace from crash:
```
20:32:03.434 [main] ERROR o.o.atlas.utilities.runtime.Command - Command execution failed.
java.lang.NullPointerException: null
	at org.openstreetmap.atlas.checks.maproulette.serializer.TaskDeserializer.deserialize(TaskDeserializer.java:57) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.checks.maproulette.serializer.TaskDeserializer.deserialize(TaskDeserializer.java:29) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at com.google.gson.TreeTypeAdapter.read(TreeTypeAdapter.java:58) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:803) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:768) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:717) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:689) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand.lambda$execute$6(MapRouletteUploadCommand.java:136) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[na:na]
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177) ~[na:na]
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133) ~[na:na]
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[na:na]
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[na:na]
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[na:na]
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[na:na]
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497) ~[na:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand.lambda$execute$7(MapRouletteUploadCommand.java:133) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at java.base/java.util.Optional.ifPresent(Optional.java:183) ~[na:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand.lambda$execute$8(MapRouletteUploadCommand.java:129) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541) ~[na:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand.execute(MapRouletteUploadCommand.java:124) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteCommand.onRun(MapRouletteCommand.java:106) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteCommand.onRun(MapRouletteCommand.java:87) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.utilities.runtime.Command.execute(Command.java:338) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.utilities.runtime.Command.run(Command.java:282) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
	at org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand.main(MapRouletteUploadCommand.java:79) ~[atlas-checks-6.1.6-SNAPSHOT-shaded.jar:na]
```

**Environment (please complete the following information):**
 - OS: Linux
 - Version 6.1.6

**Additional context**
An additional behavior was observed while debugging this that made debugging difficult. In the command above I specified -countries='NZL' but the code consumes and scan through all the country folders in the output directory when processing the request. This is fine except that if I am producing 1000s of log files in my output folder and only want to create a challenge for one country there is no need to process all the other directories.

**Comment by atiannicelli** [comment]
I'm not saying that this is the real fix, but this worked around the issue for me:
```
diff --git a/src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java b/src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java
index 7999610..c0f7648 100644
--- a/src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java
+++ b/src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java
@@ -130,7 +130,7 @@ public class MapRouletteUploadCommand extends MapRouletteCommand
             {
                 try (BufferedReader reader = FileUtility.getReader(logFile, logOutputFileType))
                 {
-                    reader.lines().filter(line -> line.trim().length() > 0).forEach(line ->
+                    reader.lines().filter(line -> line.contains().trim().length() > 0 && line.containes("features")).forEach(line ->
                     {
                         // Get Task from geojson
                         final Task task = gson.fromJson(line, Task.class);
```

**Comment by seancoulter** [comment]
See https://github.com/osmlab/atlas-checks/pull/435

**Comment by seancoulter** [comment]
Resolved by #435
